src/App.js

```diff
@@ -1,13 +1,20 @@
+import { Routes, Route } from 'react-router-dom';
 import React from 'react';
 import Calculator from './components/calculator';
 import DisplayQuote from './components/displayQuote';
+import Header from './components/Header';
+import Home from './components/Home';
 
 function App() {
   return (
     <>
+      <Header />
       <div className="app">
-        <Calculator />
-        <DisplayQuote />
+        <Routes>
+          <Route exact path="/" element={<Home />} />
+          <Route path="/calculator" element={<Calculator />} />
+          <Route path="/quote" element={<DisplayQuote />} />
```

Comment:
- [optional] Nice job generating random quotes.. Everything functions well and design is simple and direct. You can add some more design or even background design/colors to make it prettier and more appealing.